scripts/start.sh

```diff
@@ -32,30 +32,37 @@ docker exec -it $NAME scripts/upload-definitions.sh \
     "$VERSION"
 
 # import SNOMED CT
-
-docker exec -it $NAME scripts/import.sh \
-    terminologies/data/SnomedCT_ManagedServiceUS_PRODUCTION_US1000124_20230901T120000Z.zip \
-    "$VERSION" \
-    $URL \
-    "http://snomed.info/sct"
+if [ ! -e "configs/SNOMED" ]; then
+    docker exec -it $NAME scripts/import.sh \
+        terminologies/data/SnomedCT_ManagedServiceUS_PRODUCTION_US1000124_20230901T120000Z.zip \
+        "$VERSION" \
+        $URL \
+        "http://snomed.info/sct" && touch configs/SNOMED
```

Comment:
your approach will need to be adjusted because the script doesn't know anything about being in a Docker or executing in a docker.  As we discussed today, it should receive the config file as input and ingest codes using the `hapi-fhir-cli` command.